vpr/src/route/rr_graph_storage.h

```diff
@@ -185,8 +185,12 @@ class t_rr_graph_storage {
     short node_capacity(RRNodeId id) const {
         return node_storage_[id].capacity_;
     }
-    short node_cost_index(RRNodeId id) const {
-        return node_storage_[id].cost_index_;
+    RRIndexedDataId node_cost_index(RRNodeId id) const {
+        return RRIndexedDataId(node_storage_[id].cost_index_);
+    }
+
+    bool cost_index_is_invalid(RRNodeId id) const {
```

Comment:
As discussed in the VTR meeting, we can get rid of this and use the StrongID::Invalid() and/or it's bool test  functionality.